src/hdf5_back_gen.py

```diff
@@ -0,0 +1,496 @@
+#!/usr/bin/env python
+"""This module generates HDF5 backend code found in src/hdf5_back.cc"""
+import json
+import textwrap
+
+with open('../share/dbtypes.json') as f:
+    RAW_TABLE = json.load(f)
+
+V3_TABLE = set(tuple(row) for row in RAW_TABLE[897:])
+
+def list_dep(orig):
+    """Returns a list of dependent dbtypes.
+    
+    Keyword arguments: 
+    orig -- original dbtype.
```

Comment:
This should be in numpydoc format.  Checkout pyne or xonsh or the numpydoc docs :)